ecs_fargate/datadog_checks/ecs_fargate/ecs_fargate.py

```diff
@@ -157,11 +157,11 @@ def submit_perf_metrics(self, instance, container_tags, container_id, container_
 
             value_system = cpu_stats.get('system_cpu_usage')
             if value_system is not None:
-                self.rate('ecs.fargate.cpu.system', value_system, tags)
+                self.gauge('ecs.fargate.cpu.system', value_system, tags)
 
             value_total = cpu_stats.get('cpu_usage', {}).get('total_usage')
             if value_total is not None:
-                self.rate('ecs.fargate.cpu.user', value_total, tags)
+                self.gauge('ecs.fargate.cpu.user', value_total, tags)
```

Comment:
This can be dangerous if some users rely on this metric for alerting as it can trigger monitor/make them unusable.
Can we get a list of monitors that are relying on this metric and check the impact this would have? We've done in the past a couple of duplicates (submitting an extra metric called `ecs.fargate.cpu.system.gauge` with the right type) to prevent breaking changes.

Reply:
I'm not sure how it's handled in the backend but these metrics are declared gauges in `metadata.csv`

Reply:
The agent will be causing the discrepancy here, as with rate it submits the difference between values and with gauge it submits the last value. So users will see a different (but correct) value than what they're used to.
Impact will greatly depend on how many monitors are alerting on this metric, can you check and post the result in this PR? https://github.com/DataDog/devops/wiki/Whos-using-a-metric

Reply:
PR updated

Reply:
hey @FlorianVeaux , we discussed this with the team and we decided to fix the current metrics and reach out to affected customers.
cc @DylanLovesCoffee @Simwar

Reply:
Just a suggestion, could we submit the same value but with a slightly different metric name? That way we could keep compatibility (we've done this before with mongo https://github.com/DataDog/integrations-core/blob/master/mongo/metadata.csv#L201-L202)

Reply:
We discussed that possibility with PM and T2s and we preferred to reach out to every impacted customer instead of keep sending erroneous metrics. The number monitors that are alerting on this metric is quite small and can be handled.

Reply:
@ChristineTChen @FlorianVeaux could we get the PR approved before the freeze? 🙇